test/14.js

```diff
@@ -0,0 +1,17 @@
+const solution = require('../solutions/14.js');
+
+const tester = (upTo, i=2)=>{
```

Comment:
why is your test so complicated? [Look at mine it's so simple](https://github.com/seemcat/algorithm/blob/0b35dadac4ba9060fc379304f366a1b28588718e/test/8.js). You don't need to recurse.

Reply:
This is the way Royce taught me. Song told him to teach me so I could teach everyone else.

Reply:
Ok, I see the problem, no array! Will fix.

Reply:
Done!